### DISCUSSION TITLE
signature computing

### DISCUSSION THREAD

**Discussion by malosupremo** [ImageMagick/ImageMagick#3144] [Help]
Dear all,
I'd need some clarifications about the value of signature I can get in the verbose info output or with the %#, e.g:

signature: aea65c9b557d779d142f26d04abe7008bab50f1e9647faae20afdcc6c15140d5

Is this value calculated only using the image data or is the metadata also considered in this computation? I could not find an answer in the documentation.

Thanks

**Comment by snibgo** [comment]
As far as I know, the signature should be calculated from image pixel values only, and metadata should not be included.
